### PR TITLE
Changes in modules and packages presentation

### PR DESCRIPTION
**Pull request by shiramax**

### CODE REVIEW

File: docs/course_materials/exercises/07_Modules_and_packages/homework.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Functions - HomeWork 
+
+### animals.py
```
Comment (sleviim) [inline docs/course_materials/exercises/07_Modules_and_packages/homework.md:3]: add a new empty line

File: docs/course_materials/exercises/07_Modules_and_packages/homework.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+- Create a file *snake.py*
+- Create your own module with the function snake()
+- The function snake() should print "hissssss"
+- Create a ne file *main_animals.py*, import the animals module, and call the function snake()   
+
+
```
Comment (sleviim) [inline docs/course_materials/exercises/07_Modules_and_packages/homework.md:9]: redundant

File: docs/course_materials/exercises/07_Modules_and_packages/homework.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Functions - HomeWork 
+
+### animals.py
+- Create a file *snake.py*
+- Create your own module with the function snake()
+- The function snake() should print "hissssss"
+- Create a ne file *main_animals.py*, import the animals module, and call the function snake()
```
Comment (sleviim) [inline docs/course_materials/exercises/07_Modules_and_packages/homework.md:7]: ne > new

File: docs/course_materials/modules_packages.html [hunk 4]
```diff
@@ -49,10 +49,23 @@ <h1><b>
                     <h3><b>The 'import' statement</b></h3>
                     <ul>
                         <li>Allow use of other python files and libraries</li>
-                        <li>Imports: import math</li>
-                        <li>Named imports: import math as m</li>
-                        <li>Specific imports: from math import pow</li>
-                        <li>Import all: from math import * (dangerous! used only in very specific cases)</li>
+
```
Comment (sleviim) [inline docs/course_materials/modules_packages.html:52]: redundant

File: docs/course_materials/modules_packages.html [hunk 5]
```diff
@@ -49,10 +49,23 @@ <h1><b>
...
+                        <li>Imports: </li>
+                        <pre><code data-trim data-noescape class="hljs lang-python">
+                        import math
```
Comment (sleviim) [inline docs/course_materials/modules_packages.html:55]: I think it's better that all will be in 1 pre-code, and add notes with '#'

File: docs/course_materials/modules_packages.html [hunk 6]
```diff
@@ -63,21 +76,39 @@ <h3><b>Example</b></h3>
...
                 </section>
                 <section>
-                    <h3><b>Modules</b></h3>
+                    <h3><b>Make a module</b></h3>
+                    <ul>
+                      <li>
+                        To make a module, create a Python file. Then import it like any other module.
+                      </li>
+                    </ul>
                     <pre><code data-trim data-noescape class="hljs">
```
Comment (sleviim) [inline docs/course_materials/modules_packages.html:85]: hljs lang-python

File: docs/course_materials/modules_packages.html [hunk 7]
```diff
@@ -63,21 +76,39 @@ <h3><b>Example</b></h3>
...
 # utensils.py
 def eat_soup():
     return 'spoon'
+                    </code></pre>
+
+                    <pre><code data-trim data-noescape class="hljs">
```
Comment (sleviim) [inline docs/course_materials/modules_packages.html:91]: hljs lang-python

File: docs/course_materials/modules_packages.html [hunk 8]
```diff
@@ -63,21 +76,39 @@ <h3><b>Example</b></h3>
...
 
 # main.py (option 1)
 import utensils
 print(utensils.eat_soup())
+                    </code></pre>
+                    <pre><code data-trim data-noescape class="hljs">
```
Comment (sleviim) [inline docs/course_materials/modules_packages.html:97]: hljs lang-python

**Review by sleviim — CHANGES_REQUESTED** [review]

File: docs/course_materials/modules_packages.html (hunk 5 above)
Comment (shiramax) [reply docs/course_materials/modules_packages.html:55]: disagree, because it's shouldn't be in the same file, it's different options/

Comment (sleviim) [reply docs/course_materials/modules_packages.html:54]: we can see the feedbacks after the class :)

**Review by sleviim — APPROVED** [review]

File: docs/course_materials/modules_packages.html [hunk 9]
```diff
@@ -49,35 +49,65 @@ <h1><b>
...
                     <ul>
                         <li>Allow use of other python files and libraries</li>
-                        <li>Imports: import math</li>
-                        <li>Named imports: import math as m</li>
-                        <li>Specific imports: from math import pow</li>
-                        <li>Import all: from math import * (dangerous! used only in very specific cases)</li>
+                        <li>Imports: </li>
+                        <pre><code data-trim data-noescape class="hljs lang-python">
+                        import math
+                        </code ></pre>
```
Comment (sleviim) [inline docs/course_materials/modules_packages.html:55]: trailing space after the code

File: docs/course_materials/modules_packages.html [hunk 10]
```diff
@@ -49,35 +49,65 @@ <h1><b>
...
+                        <li>Named imports: </li>
+                        <pre><code data-trim data-noescape class="hljs lang-python">
+                          import math as m
+                        </code ></pre>
```
Comment (sleviim) [inline docs/course_materials/modules_packages.html:59]: same

File: docs/course_materials/modules_packages.html [hunk 11]
```diff
@@ -49,35 +49,65 @@ <h1><b>
...
+                        <li>Specific imports: </li>
+                        <pre><code data-trim data-noescape class="hljs lang-python">
+                        from math import pow
+                        </code ></pre>
```
Comment (sleviim) [inline docs/course_materials/modules_packages.html:63]: same

File: docs/course_materials/modules_packages.html [hunk 12]
```diff
@@ -49,35 +49,65 @@ <h1><b>
...
+                        <li>Import all: (dangerous! used only in very specific cases)</li>
+                        <pre><code data-trim data-noescape class="hljs lang-python">
+                        from math import *
+                        </code ></pre>
```
Comment (sleviim) [inline docs/course_materials/modules_packages.html:67]: same

**Review by sleviim — CHANGES_REQUESTED** [review]

**Review by sleviim — APPROVED** [review]

**Review by sleviim — APPROVED** [review]

**Pull request merged**